src/main/flight/mixer_init.c

```diff
@@ -322,6 +331,19 @@ void mixerInitProfile(void)
         }
     }
 #endif
+
+#ifdef USE_RPM_LIMITER
+    mixerRuntime.rpmLimiterExpectedThrottleLimit = 1.0f;
```

Comment:
You should try and figure out how to store the value of this once you disarm. That way the learned limit won't have to be relearned every time you plug in a new battery.